src/Catalog/Telemetry/TelemetryConstants.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public static class TelemetryConstants
+    {
+        public const string CatalogIndexReadDurationSeconds = "CatalogIndexReadDurationSeconds";
+        public const string CatalogIndexWriteDurationSeconds = "CatalogIndexWriteDurationSeconds";
+        public const string ContentBaseAddress = "ContentBaseAddress";
+        public const string ContentLength = "ContentLength";
+        public const string CreatedPackagesCount = "CreatedPackagesCount";
+        public const string CreatedPackagesSeconds = "CreatedPackagesSeconds";
+        public const string DeletedPackagesCount = "DeletedPackagesCount";
+        public const string DeletedPackagesSeconds = "DeletedPackagesSeconds";
+        public const string Destination = "Destination";
+        public const string EditedPackagesCount = "EditedPackagesCount";
+        public const string EditedPackagesSeconds = "EditedPackagesSeconds";
+        public const string HttpHeaderDurationSeconds = "HttpHeaderDurationSeconds";
+        public const string Id = "Id";
+        public const string LoopSeconds = "LoopSeconds";
+        public const string Method = "Method";
+        public const string PackageDownloadSeconds = "PackageDownloadSeconds";
+        public const string ProcessGraphsSeconds = "ProcessGraphsSeconds";
+        public const string ProcessPackageDelete = "ProcessPackageDelete";
+        public const string ProcessPackageDetails = "ProcessPackageDetails";
+        public const string RegistrationDeltaCount = "RegistrationDeltaCount";
+        public const string Size = "Size";
```

Comment:
I could not find the TelemetryConstants.Size being used. Is the size referring to a file size?